firebase-firestore/CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 # Unreleased
-
+* [feature] Implemented an optimization in the local cache synchronization logic that reduces the number of billed document reads when documents were deleted on the server while the client was not actively listening to the query (e.g. while the client was offline).
```

Comment:
Please add `(GitHub [#4702](//github.com/firebase/firebase-android-sdk/pull/4982){: .external})`

Reply:
what does this `{: .external}` means? do we also have `{: .internal}` or sth like that?

Reply:
> what does this `{: .external}` means? do we also have `{: .internal}` or sth like that?

I honestly don't know. I just copied it from other entries below. I think it goes through some kind of converter to generate the ultimate release notes web page.